packages/ts/react-signals/test/ValueSignal.spec.tsx

```diff
@@ -137,6 +137,32 @@ describe('@vaadin/hilla-react-signals', () => {
       expect(valueSignal.value).to.equal('baz');
     });
 
+    it('should accept a callback after replace', (done) => {
```

Comment:
Just asking due to my lack of knowledge:
How passing the `done` to `.result.then(done)` asserts that it is called?

Reply:
Mocha itself expects `done` to be called if declared in the function. So, if it's not called, the test waits for some timeout and then fails.